build error in database.c

**Describe the bug**
building current master on osx results in:
```
[ 18%] Building C object src/CMakeFiles/lib_darktable.dir/common/database.c.o
/Users/martinstraeten/src/darktable/src/common/database.c:2810:100: error:
      format specifies type 'long long' but the argument has type 'guint64'
      (aka 'unsigned long') [-Werror,-Wformat]
  ...maintenance suggested, %" PRId64 " bytes to free.\n", calc_size);
                            ~~~~~~~~~                      ^~~~~~~~~
/Users/martinstraeten/src/darktable/src/common/database.c:2815:95: error:
      format specifies type 'long long' but the argument has type 'guint64'
      (aka 'unsigned long') [-Werror,-Wformat]
  ...maintenance] maintenance done, %" PRId64 " bytes freed.\n", calc_size);
                                    ~~~~~~~~~                    ^~~~~~~~~
2 errors generated.
```
**Platform (please complete the following information):**
 - Darktable Version: current master
 - OS: osx
 - current macports

--- a/src/common/database.c
+++ b/src/common/database.c
@@ -2807,12 +2807,12 @@ void dt_database_maybe_maintenance(const struct dt_database_t *db, const gboolea
       || (data_free_percentage >= freepage_ratio))
   {
     const guint64 calc_size = (main_free_count*main_page_size) + (data_free_count*data_page_size);
-    dt_print(DT_DEBUG_SQL, "[db maintenance] maintenance suggested, %" PRId64 " bytes to free.\n", calc_size);
+    dt_print(DT_DEBUG_SQL, "[db maintenance] maintenance suggested, %" G_GUINT64_FORMAT " bytes to free.\n", calc_size);
 
     if(force_maintenance || _ask_for_maintenance(has_gui, closing_time, calc_size))
     {
       _dt_database_maintenance(db);
-      dt_print(DT_DEBUG_SQL, "[db maintenance] maintenance done, %" PRId64 " bytes freed.\n", calc_size);
+      dt_print(DT_DEBUG_SQL, "[db maintenance] maintenance done, %" G_GUINT64_FORMAT " bytes freed.\n", calc_size);
     }
   }
 }

